customer-service/omnichannel/smart-assist-bot/README.md

```diff
@@ -1,14 +1,158 @@
 # Build a custom smart assist bot
 
+### Getting Started
+This sample code contains custom smart bot implementation for suggesting KB articles and for Natural Language Understanding using [LUIS](https://www.luis.ai/home).
+
+### 1. Deploying a new Smart Assist Bot
+
+To develop a new web application based smart-assist bot, follow the steps mentioned below.
+
+- Download the github sample from [this page](https://github.com/microsoft/Dynamics365-Apps-Samples)
+- Go to directory **Dynamics365-Apps-Samples\customer-service\omnichannel\smart-assist-bot\SmartAssistBot**
+- Open the solution file [SmartBot.sln](https://github.com/microsoft/Dynamics365-Apps-Samples/blob/master/customer-service/omnichannel/smart-assist-bot/SmartAssistBot/CoreBot.sln) in Microsoft Visual Studio 2017 or above
+
+This setup broadly contains the following steps.
+
+1) Update the values in appsettings.json (SmartAssistBot/appsettings.json) file<br />
+2) Publish the webapp to Azure<br />
+3) Update the Messaging endpoint in Bot channels registration<br />
+4) Enable Teams channel<br />
+5) Test your Bot online<br />
```

Comment:
Add a step to link back to bot configuration for Smart assist.

Reply:
Done !